src/calls-cost.js

```diff
@@ -39,6 +40,85 @@
  * @returns {CallsResponse}  - Processed information
 */
 
-function callsCost(calls) { }
+function callsCost(calls) {
+
+  // calls : Array<Object>
+
+  let processedCalls = [];
+  let totalDuration = 0;
+  let numberOfCalls = 0;
+  let totalPay = 0;
+
+  calls.forEach(call => {
+
+    switch (call.type) {
+
+      case 'International':
+
+        let costInternational = 0;
+
```

Comment:
In order to improve the readability of your code, only add whitespace to separate logic blocks.